data/DataType/boolean_java.yaml

```diff
@@ -4,4 +4,4 @@ integer-max: '1'
 integer-signed: false
 literals:
   - 'true'
-  - 'false'
+  - 'false'
```

Comment:
no new line character at the end of file (new-line-at-end-of-file)

*Origin: YAMLLintBear, Section: `yaml`.*